### DISCUSSION TITLE
shortcut doesnt control volume as it should be

### DISCUSSION THREAD

**Discussion by denizeylulparlak** [MonitorControl/MonitorControl#1566] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

i attend a shortcut for volume control but appareantly it deosnt work (its animations seen on screen but it doesnt work)

### Steps to reproduce

attend a shotcut for volume control its animation will seen on screen wehn u try to use but volume doesnt change 

### Expected behavior

can control volume

### Environment Information (please complete the following information)

```markdown
- macOS version:v13.2.1
- Mac model:macbook pro m1 pro base model 14 
- MonitorControl version:v4.2
- Monitor(s):Dell UltraSharp U2723QE
- Apple Silicon/M1 (yes or no):yes
```

**Comment by daaain** [comment]
Same issue with 4.2.0 using a Dell S3423DWC on Mac OS 14.4.1 with an MBP M2 Max via USB C / DisplayPort.

The volume control shows up, but it stays on full volume.

**Comment by diasthiago** [comment]
Same issue with 4.2.0 using a Dell U2422H on Mac OS 14.5 with an MBP M1 Pro via HDMI → HDMI.

**Comment by Glazomer** [comment]
Same here:
- macOS version: v14.4
- Mac model: macbook pro m1 pro max 16
- MonitorControl version: v4.2
- Monitor(s): Acer XR342CPK
- Apple Silicon/M1 (yes or no): yes

**Comment by waydabber** [comment]
Sadly not all displays support DDC volume control and some displays support it on some ports only. You can try alternative apps like https://github.com/waydabber/BetterDisplay, https://lunar.fyi or m1ddc to check whether this is a generic DDC control issue or a MonitorControl specific problem.

**Reply by daaain** [reply]
I tried with Lunar, same issue, and then after a bit more testing I realised the volume control does actually work with both MonitorControl and Lunar, but only for the built-in speakers and I wanted to control the audio jack out, which stays the same even if I change the volume using the OSD controls (that I just noticed) 🤦 so, the apps and the control protocol are fine, but Dell didn't make the audio out possible to control.
